readalongs/cli.py

```diff
@@ -101,10 +153,12 @@ def cli():
     help="Use ReadAlong-Studio configuration file (in JSON format)",
 )
 @click.option(
-    "-C",
-    "--closed-captioning",
-    is_flag=True,
-    help="Export sentences to WebVTT and SRT files",
+    "-o",
+    "--output-formats",
+    type=click.Choice(SUPPORTED_OUTPUT_FORMATS.keys()),
+    multiple=True,
+    help="The output file formats to export to. By default the text is exported as XML and alignments are exported as SMIL. "
+    + f"Other formats include:\b \n\n{SUPPORTED_OUTPUT_FORMATS_DESC}",
```

Comment:
This is mostly good, but on my terminal `readalongs align -h` add a lot of whilespace here, but a full blank line between each format. That seems excessive to me. I'd prefer something like
```
                                  alignments are exported as SMIL. Other
                                  formats include:
                                   - eaf - ELAN file
                                   - html - Single-file, offline HTML
                                   - srt - SRT subtitle
                                   - TextGrid - Praat TextGrid
                                   - vtt - WebVTT subtitle
                                   - xhtml - Simple XHTML
```

Reply:
good point, I'll try and take away that extra white space

Reply:
hm. if I do just a single \n it produces this 
![image](https://user-images.githubusercontent.com/14815080/140199183-4384c374-9c50-4439-86c7-f7afca23e6b1.png)
which is worse. ideas?

Reply:
![image](https://user-images.githubusercontent.com/14815080/140199284-14cef792-b440-4f0b-b4a7-b39e98304722.png)
 with pipes?

Reply:
Darn, I just tested and \b doesn't solve it either. How about `eaf (ELAN file), html (Single-file, offline HTML), srt (SRT subtitle), ...`

Reply:
sure, that would work

Reply:
done